config/locales/en.yml

```diff
@@ -127,22 +127,24 @@ en:
     assume_roles_heading: "Switch Roles"
```

Comment:
I know these are left over from other changes but there are still some `""` living on in this file, should they also be removed in this PR or another?

Reply:
I haven't done the whole file, just the area I was touching